docs/upgrade.rst

```diff
@@ -9,6 +9,40 @@ UNRELEASED
 End of life Python 3.3 and Django 1.8 are no longer supported. These versions
 of Python and Django no longer receive security patches.
 
+0.9.1
+=====
+
+Support for django 2.1 has been added, support for django<1.11 is removed.
+Support for deprecated South has been removed.
+
+Some updates have been made on the documentation such as:
+
+- Misc updates in the docs (formatting)
+- Added italian translation
+- Remove unnecessary db_index=True from doc examples
```

Comment:
Would be nice to use double backticks (``db_index=True``) so that code is formatted as such.